new-log-viewer/src/typings/worker.ts

```diff
@@ -64,8 +66,20 @@ type WorkerReqMap = {
         cursor: CursorType,
         decoderOptions?: DecoderOptionsType
     },
+    [WORKER_REQ_CODE.QUERY_LOG]: {
+        searchString: string,
+        isRegex: boolean,
+        isCaseSensitive: boolean,
+    },
 };
 
+type ChunkResultType = {
```

Comment:
Depending on the discussion at https://github.com/y-scope/yscope-log-viewer/pull/80/files#r1791414114 , we shall name this as `queryResultType` or `searchResultType`.